app/controllers/concerns/course/user_bulk_management_concern.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+module Course::UserBulkManagementConcern
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :authorize_show!, only: [:students, :staff, :requests, :invitations]
+    before_action :authorize_edit!, only: [:update, :destroy]
+  end
+
+  def update # :nodoc:
+    if update_course_user(course_user_params)
+      success = t('course.users.update.success', role: t("course.users.role.#{@course_user.role}"))
+      redirect_to update_redirect_path, success: success
+    else
+      redirect_to update_redirect_path, danger: @course_user.errors.full_messages.to_sentence
+    end
+  end
+
+  def destroy # :nodoc:
+    if @course_user.destroy
+      success = t('course.users.destroy.success', role: @course_user.role,
+                  email: @course_user.user.email)
```

Comment:
@weiqingtoh I think this was unintentional. The `email` should be aligned with `role`.